release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -967,6 +967,12 @@ You cannot upgrade {op-system-base} 7 compute machines to {op-system-base} 8. Yo
 [id="ocp-4-10-console-storage-bug-fixes"]
 ==== Console Storage Plug-in
 
+* Previously, a loading prompt was not present while the PV’s were being provisioned and the capacity was 0 TiB. Consequently, this created a confusing scenario for the users. With this update, a loader is added for the loading state which provides details to the user such as: including provisions of the PVs, determines the capacity, or encounters any errors in the process. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1928285[*BZ#1928285*])
```

Comment:
s/PV's/persistent volumes (PVs)

    `0` TiB, which created a confusing scenario.

state, which provides

Are the listed items after "such as" menu items? The structure seems a bit awkward and I don't understand what is meant there.

Reply:
Updated. lmk what you think.